client/guidestours/index.js

```diff
@@ -175,6 +175,7 @@ export default class GuidesTours extends Component {
 	}
 
 	quit() {
+		//TODO: should we dispatch a showGuidesTour action here instead?
```

Comment:
We haven't `connect`ed GuidesTours yet, so I think I'll leave that for another PR, in case the components change. Leaving the TODO though :)